### ISSUE TITLE
Datascience Theme Comments

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/quantecon-book-theme#133]
https://github.com/QuantEcon/lecture-datascience.myst/issues/119

@DrDrij thoughts?

**Comment by DrDrij** [comment]
@mmcky I reckon it's pretty useful functionality. Almost like the minimap in code editors. I'd be happy to implement.
This would include:
- Make the whole in-page-toc sticky (not just 'back to top')
- Add active highlight to toc items depending on page scroll

**Comment by mmcky** [comment]
Yeah it is pretty good. Now that we are inheriting from `sphinx-book-theme` are we just overriding the styles? Or do we need to implement this in our context?

**Comment by DrDrij** [comment]
I'll take a closer look. There might be a tiny little work to refactor the markup so that it can use the JS/CSS from SBT. 
Hopefully as you have suggested all we'll need then is some QBT styling.
Shall I implement this across the `quantecon-book-theme` ?

**Comment by mmcky** [comment]
Yeah let's see what it looks like. I think the animation and navigation is nice but John might like the simplicity of what we have. So let's make a PR and get comments :-)
